src/bitcoin-cli.cpp

```diff
@@ -867,6 +874,84 @@ static void GetWalletBalances(UniValue& result)
     result.pushKV("balances", balances);
 }
 
+/**
+ * ParseGetInfoResult takes in -getinfo result in UniValue object and parses it
+ * into a user friendly UniValue string to be printed on the console.
+ * @param[out] result  Reference to UniValue result containing the -getinfo output.
+ */
+static void ParseGetInfoResult(UniValue& result)
+{
+    if (!find_value(result, "error").isNull()) return;
+
+    std::string RESET, GREEN, BLUE, YELLOW, MAGENTA, CYAN;
+
+#ifndef WIN32
+    if (isatty(fileno(stdout))) {
+      // Only print colored text if OS is not WIN32 and stdout is connected to a terminal.
+      RESET = "\x1B[0m";
+      GREEN = "\x1B[32m";
+      BLUE = "\x1B[34m";
+      YELLOW = "\x1B[33m";
+      MAGENTA = "\x1B[35m";
+      CYAN = "\x1B[36m";
+    }
+#endif
+
+    std::string result_string = strprintf("%sChain: %s%s\n", BLUE, result["chain"].getValStr(), RESET);
+    result_string += strprintf("Blocks: %s\n", result["blocks"].getValStr());
+    result_string += strprintf("Headers: %s\n", result["headers"].getValStr());
+    result_string += strprintf("Verification progress: %s\n", result["verificationprogress"].getValStr());
```

Comment:
nit: instead of `verificationprogress: 0.9999...` or `verificationprogress: 0.9977977097643339`, this change shows `verificationprogress: 100%` and `verificationprogress: 99.78%` respectively 

```suggestion
    result_string += strprintf("Verification progress: %.2f%%\n", result["verificationprogress"].get_real() * 100);
```

Reply:
Thanks a lot for reviewing! 

I actually considered changing it to 2 decimal places but was on the fence for it. My concern is that currently `0.01%` of the current height(`686181`) is about 6 blocks and displaying only 2 decimal places could confuse the user into thinking that their node is stuck. Additionally, as time goes on and the best height increases, I feel that it would be better to have a more granular value for verification progress.

What do you think? If the others also feel that having 2 decimal places is better I would change it! 😄

Reply:
I think Isilva01's proposal was primarily to change to per-cent unit, and the number of decimal places is just an example. Personally I'd also prefer showing in %; how about with 4 decimal places? With that, even a total block height of 1000000 (still years away) each single block verified results in a change of the verification progress display.

Reply:
Yeah that makes sense! % would be better than fractional from. Implemented this in c5625c2. Thanks!